_posts/2020-10-19-Software-Synth.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Software Synth in Jamulus"
+lang: "en"
+author: "niebert"
+heading: "Software Synth in Jamulus"
+---
+This post will discuss how to setup a software synth in Jamulus.
+<!--more-->
+
+## Linux
+We will elaborate the concept of playing a software synth on Linux as example and transfer that to other operating systems (e.g. Windows, MacOSX).
+* **(Hammond Organ Synth)** As an example we talk open source software synth [setBfree](https://github.com/pantherb/setBfree), which is a hammond organ emulator that outputs through JACK on Linux.
+* **(MIDI Keyboard)** The MIDI keyboard is used to generate the key pressed events, that are transmitted through a USB-MIDI interface to Linux system and the Synthesizer [setBfree](https://github.com/pantherb/setBfree) generates the corresponding tone for the pressed key. In general the MIDI keyboard can also be replaced by a digital wind midi controller, that does the same but the tone is dependent on how the musician blows air into the digital wind controller.
```

Comment:
* "to Linux system" -> "to a Linux system" (not "a" or "the")
* "Synthesizer [setBfree](https://github.com/pantherb/setBfree)" -> "[setBfree](https://github.com/pantherb/setBfree) synthesizer"

Question: are we sure we want to talk about such a specific MIDI instrument here (the wind one)? I think it's not very useful since any MIDI controller would be seen as the same "MIDI events provider" by the software and I assume we don't want to talk about specifics (MIDI guitar, MIDI saxophone,...).

Reply:
I would leave it as is for now. These articles are supposed to be user-contributed, not official documentation so I don't think we need to spend much energy on them.

Reply:
Ok. If you think that’s the case we can soon merge this. We must keep in mind that also community contributions should not have too many errors, spelling mistakes etc. (That’s good for the user and SEO).